ui/raidboss/data/05-shb/trial/hades-ex.js

```diff
@@ -274,21 +306,241 @@
+            fr: 'Tankbuster sur VOUS',
+          };
+        }
+        if (data.role == 'healer') {
+          return {
+            en: 'Buster on ' + data.ShortName(matches[1]),
+            de: 'Tankbuster auf ' + data.ShortName(matches[1]),
+            fr: 'Tankbuster sur ' + data.ShortName(matches[1]),
+          };
+        }
+        return {
+          en: 'Away from ' + data.ShortName(matches[1]),
+        };
+      },
+    },
+    {
+      id: 'HadesEx Megiddo Flame',
+      regex: / 14:47CD:Ascian Prime's Shade starts using Megiddo Flame on Ascian Prime's Shade\./,
+      suppressSeconds: 1,
+      infoText: {
+        en: 'Healer Stacks',
+      },
+    },
+    {
+      id: 'HadesEx Shadow Flare',
+      regex: / 14:47D0:Ascian Prime's Shade starts using Shadow Flare on Ascian Prime's Shade\./,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      infoText: {
+        en: 'aoe',
+        de: 'AoE',
+        fr: 'Dégâts de zone',
+      },
+    },
+    {
+      id: 'HadesEx Captivity',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:0078:/,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      infoText: {
+        en: 'Get Out',
+      },
+    },
+    {
+      id: 'HadesEx Aetherial Gaol',
+      regex: / 03:\y{ObjectId}:Added new combatant Aetherial Gaol\./,
+      infoText: {
+        en: 'Break Aetherial Gaol',
+      },
+    },
+    {
+      id: 'HadesEx Dark Flame',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:0064:/,
+      suppressSeconds: 1,
+      condition: function(data) {
+        return data.role == 'tank' || data.role == 'healer';
+      },
+      infoText: function(data, matches) {
+        if (matches[1] == data.me) {
+          return {
+            en: 'Stack on YOU',
+          };
+        }
+        return {
+          en: 'Stack With Healer',
+        };
+      },
+    },
+    {
+      id: 'HadesEx Dark Freeze',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:0078:/,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      infoText: {
+        en: 'Ice on YOU',
+      },
+    },
+    {
+      id: 'HadesEx Nether Blast',
+      regex: / 1B:\y{ObjectId}:(\y{Name}):....:....:008B:/,
+      condition: function(data, matches) {
+        return data.me == matches[1];
+      },
+      run: function(data) {
+        data.netherBlast = true;
+      },
+      alertText: {
+        en: 'Puddles on YOU',
+      },
+    },
+    {
+      id: 'HadesEx Bident',
+      regex: / 14:47E3:Hades starts using Bident on Hades\./,
+      condition: function(data) {
+        return data.netherBlast == false;
+      },
+      infoText: {
+        en: 'Healer Stacks',
+      },
+    },
+    {
+      id: 'HadesEx Shadow Stream',
+      regex: / 14:47EA:Hades starts using Shadow Stream on Hades\./,
+      alertText: {
+        en: 'Out Of Middle',
```

Comment:
I would say "Go sides".  Ideally we can look where the orbs are and say "Left and front" or something?